apiserver/applicationoffers/applicationoffers.go

```diff
@@ -29,8 +28,7 @@ type environFromModelFunc func(string) (environs.Environ, error)
 // implementation of the api end point.
 type OffersAPI struct {
 	BaseAPI
-	dataDir    string
-	getEnviron environFromModelFunc
+	dataDir string
```

Comment:
I know it isn't in this change, but I don't really understand the split between OffersAPI and BaseAPI.